docs/errors/E208.md

```diff
@@ -0,0 +1,32 @@
+# E208: cannot access private identifier outside class
+
+The following code is accessing a private identifier outside
+the class.
+
+    class C {
+        #x = 10;
+    }
+
+    function f() {
+      this.#x = 20;
+    }
```

Comment:
Minor: A better example would be accepting an object as a parameter, not using `this`.